packages/reapit-connect-management/src/components/ui/users/edit-user-group.tsx

```diff
@@ -0,0 +1,170 @@
+  groupIds: {
+    name: 'userId',
+    label: 'Group Members',
+  },
+}
+
+export const prepareGroupOptions: (data: GroupModel[]) => SelectOption[] = data =>
+  data.map((UserGroupGroup: UserModel) => {
+    const { id, name } = UserGroupGroup
+
+    return {
+      label: name,
+      value: id,
+    } as SelectOption
+  })
+
+const addUserToGroup = async (id: string, userId: string) => {
+  const addUserRes = await addMemberToGroup({ id, userId })
+  return addUserRes
+}
+
+const removeUserFromGroup = async (id: string, userId: string) => {
+  const removeUserRes = await removeMemberFromGroup({ id, userId })
+  return removeUserRes
+}
+
+export const onHandleSubmit = (
+  handleOnClose: () => void,
+  onRefetchData: () => void,
+  mutate: () => void,
+  editingUserGroup?: GroupModel,
+  initMembers?: string[],
+) => async (params: UpdateUserGroupModel) => {
+  const id = editingUserGroup?.id || ''
+  const userId = params.userId
+  const newUserIds = difference(userId, initMembers)
+  const removeUser = difference(initMembers, userId)
+
+  for (const user of newUserIds) {
+    const addUserRes = await addUserToGroup(id, user)
+    if (addUserRes) {
+      return notification.error({
+        message: addUserRes.description || toastMessages.FAILED_TO_EDIT_USER_GROUP,
+        placement: 'bottomRight',
+      })
+    }
+  }
+
+  for (const user of removeUser) {
+    const removeUserRes = await removeUserFromGroup(id, user)
+    if (removeUserRes) {
+      return notification.error({
+        message: removeUserRes.description || toastMessages.FAILED_TO_EDIT_USER_GROUP,
+        placement: 'bottomRight',
+      })
+    }
+  }
+
+  notification.success({
+    message: toastMessages.CHANGES_SAVE_SUCCESS,
+    placement: 'bottomRight',
+  })
+  handleOnClose()
+  mutate()
+  return onRefetchData()
+}
+
+export const UpdateUserGroupModal: React.FC<UpdateUserGroupModalProps> = ({
+  editingUserGroup,
+  setEditingUserGroup,
+  onRefetchData,
+}) => {
+  const id = editingUserGroup?.id
+  const handleOnClose = () => setEditingUserGroup(undefined)
+  const { groupIds } = formFields
+
+  const { data }: any = useSWR(`${URLS.USERS}`)
+
+  const { data: groupMembers, mutate }: any = useSWR(id ? `${URLS.USERS_GROUPS}/${id}/members` : null)
+
+  if (!editingUserGroup) return null
+  if (!data) return <Loader />
+  if (!groupMembers) return <Loader />
+  const { _embedded: listUserGroup } = data
+  const { _embedded: listUserGroupMember } = groupMembers
+  const initMembers = listUserGroupMember.map((member: UserModel) => member.id)
+  const onSubmit = onHandleSubmit(handleOnClose, onRefetchData, mutate, editingUserGroup, initMembers)
+
+  const UserGroupGroupOptions = prepareGroupOptions(listUserGroup)
+
+  return (
+    <ModalV2
+      visible={!!editingUserGroup}
+      destroyOnClose={true}
+      onClose={handleOnClose}
+      title={`Editing ${editingUserGroup.id}`}
+      zIndex={90}
+    >
+      <p className="hepler-text">
```

Comment:
Helper text should be wrapped in an `<i>` tag

Also, helper text is `The list below contains all available member groups for your organisation. You can manage users associated to each group by selecting ‘Manage’.` see: https://github.com/reapit/foundations/issues/3260